x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/oidc/OpenIdConnectAuthenticator.java

```diff
@@ -531,8 +531,8 @@ private void handleTokenResponse(HttpResponse httpResponse, ActionListener<Tuple
                 final AccessToken accessToken = oidcTokens.getAccessToken();
                 final JWT idToken = oidcTokens.getIDToken();
                 if (LOGGER.isTraceEnabled()) {
-                    LOGGER.trace("Successfully exchanged code for ID Token: [{}] and Access Token [{}]",
-                        idToken, accessToken);
+                    LOGGER.trace("Successfully exchanged code. ID Token [{}] and Access Token [{}]",
```

Comment:
We should print a trimmed version of the access token ( I suggest 2 first and 2 last chars ) for correlation on troubleshooting. ID Token can be printed as it was.